static/js/components/CourseCard.js

```diff
@@ -0,0 +1,65 @@
+// @flow
+/* global SETTINGS:false */
+import React from "react"
+import Dotdotdot from "react-dotdotdot"
+
+import { availabilityLabel, minPrice } from "../lib/courses"
+import { embedlyThumbnail } from "../lib/url"
+import {
+  CAROUSEL_IMG_WIDTH,
+  CAROUSEL_IMG_HEIGHT,
+  platformLogoUrls
+} from "../lib/constants"
+
+import type { Course } from "../flow/discussionTypes"
+
+type CardProps = {|
+  course: Course,
+  setShowCourseDrawer: Function
+|}
+
+const CourseCard = ({ course, setShowCourseDrawer }: CardProps) => (
+  <div
+    className="course-card"
+    onClick={() => setShowCourseDrawer({ courseId: course.id })}
+  >
+    <div className="card-contents">
+      <div className="cover-image">
+        <img
+          src={embedlyThumbnail(
+            SETTINGS.embedlyKey,
+            course.image_src || "",
+            CAROUSEL_IMG_HEIGHT,
+            CAROUSEL_IMG_WIDTH
+          )}
+          height={CAROUSEL_IMG_HEIGHT}
+        />
+      </div>
+      <div className="row course-title">
+        <Dotdotdot clamp={2}>{course.title}</Dotdotdot>
+      </div>
+      <div className="row topics">
+        {course.topics.length > 0
+          ? course.topics.slice(0, 3).map(topic => (
+            <div className="topic" key={topic.name}>
+              {topic.name}
+            </div>
+          ))
+          : null}
+      </div>
+      <div className="row availability">
+        {availabilityLabel(course.availability)}
+      </div>
+      <div className="row platform">
+        <img
+          className="course-platform"
+          src={platformLogoUrls[course.platform]}
```

Comment:
Could we have an `alt` attribute here?

Reply:
I added one here and then went ahead and added one to the 'cover image' above as well